### PR TITLE
My portfolio

### PR DESCRIPTION
**Pull request by oluwajuwon8**
Simple portfolio example derived from figma template.

### CODE REVIEW

**Review by AshakaE — COMMENTED** [review]
### Hi @oluwajuwon8 🙋 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request ❌ 

You have submitted a project, that has failed linter checks

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,2 +1,45 @@
...
+## Built With
+
+- HTML 
+- CSS STYLE
+
+### setup 
+To get this project in you local machine, follow the following steps:
+- Open terminal 
+- Clone ...
+- Cd in project folder
```
Comment (elisha2kyakpo1) [inline README.md]: - [ ] This part you have to provide more information. When you just write clone...., a user who is not familiar with git and GitHub will find it difficult to get your project.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      <h4>Hey There.</h4>
+      <h4>I'm Azeezat</h4>
+      <h3>
+        I'm a software Developer! <br />
+        <br />
+        I can help you build a product,feature or website. Look through some of
+        my work and Experience. if you like what you see and have a project dont
+        hesitate to contact me.
+      </h3>
+    </section>
```
Comment (elisha2kyakpo1) [inline index.html]: - [ ] The h3 tag( I'm a Software Developer) should be a paragraph tag

- [ ] This area should have a background image. See the original image below:
port

File: index.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        <li>
+          <a href="#"><i class="fab fa-linkedin"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fab fa-youtube"></i></a>
+        </li>
+        <li>
+          <a href="#"><i class="fab fa-medium"></i></a>
+        </li>
+      </ul>
```
Comment (elisha2kyakpo1) [inline index.html]: - [ ] If we look clearly, the social icons should be part of the dark(black) background color(section on top)

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi, @oluwajuwon8,

Great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

- [ ] Pull Request should have a short summary describing introduced changes in more detail. In this case, you are not mentioning what you did.
summary


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Branch name should describe a feature or action (e.g. 'created_sidebar' or 'added_comments_section').

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by george-shammar — APPROVED** [review]
Hi @oluwajuwon8 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**